sdk/servicebus/service-bus/src/core/messageSender.ts

```diff
@@ -293,14 +292,14 @@ export class MessageSender extends LinkEntity<AwaitableSender> {
    * Initializes the sender session on the connection.
    */
   public async open(
-    options?: AwaitableSenderOptions,
-    abortSignal?: AbortSignalLike
+    options: AwaitableSenderOptions | undefined,
+    operationOptions: OperationOptionsBase
   ): Promise<void> {
     try {
       if (!options) {
         options = this._createSenderOptions(Constants.defaultOperationTimeoutInMs);
       }
-      await this.initLink(options, abortSignal);
+      await this.initLink(options, operationOptions || {});
```

Comment:
operationOptions can't be null/undefined anymore (fro your change above)

Reply:
Nice catch

Reply:
Done